exercises/practice/triangle/triangle_test.go

```diff
@@ -5,29 +5,29 @@ import (
 )
 
 type testCase struct {
-	want    Kind
-	a, b, c float64
+	description string
+	expected    Kind
+	a, b, c     float64
 }
 
-// basic test cases
-var testData = []testCase{
-	{Equ, 2, 2, 2},    // same length
-	{Equ, 10, 10, 10}, // a little bigger
-	{Iso, 3, 4, 4},    // last two sides equal
-	{Iso, 4, 3, 4},    // first and last sides equal
-	{Iso, 4, 4, 3},    // first two sides equal
-	{Iso, 10, 10, 2},  // again
-	{Sca, 3, 4, 5},    // no sides equal
-	{Sca, 10, 11, 12}, // again
-	{Sca, 5, 4, 2},    // descending order
-	{Sca, .4, .6, .3}, // small sides
-	{Sca, 5, 4, 6},    // 2a == b+c looks like equilateral, but isn't always.
-	{Sca, 6, 4, 5},    // 2a == b+c looks like equilateral, but isn't always.
-	{NaT, 0, 0, 0},    // zero length
-	{NaT, 3, 4, -5},   // negative length
-	{NaT, 1, 1, 3},    // fails triangle inequality
-	{NaT, 2, 5, 2},    // another
-	{NaT, 7, 3, 2},    // another
+var testCases = []testCase{
+	{description: "same length (2)", expected: Equ, a: 2, b: 2, c: 2},
+	{description: "same length (10)", expected: Equ, a: 10, b: 10, c: 10},
+	{description: "b = c = 4", expected: Iso, a: 3, b: 4, c: 4},
+	{description: "a = c = 4", expected: Iso, a: 4, b: 3, c: 4},
+	{description: "a = b = 4", expected: Iso, a: 4, b: 4, c: 3},
+	{description: "a = b = 10", expected: Iso, a: 10, b: 10, c: 2},
+	{description: "no sides equal (3, 4, 5)", expected: Sca, a: 3, b: 4, c: 5},
+	{description: "no sides equal (10, 11, 12)", expected: Sca, a: 10, b: 11, c: 12},
+	{description: "no sides equal (5, 4, 2)", expected: Sca, a: 5, b: 4, c: 2},
+	{description: "no sides equal (0.4, 0.6, 0.3)", expected: Sca, a: .4, b: .6, c: .3},
+	{description: "no sides equal (5, 4, 6) | 2a=b+c", expected: Sca, a: 5, b: 4, c: 6},
+	{description: "no sides equal (6, 4, 5) | 2c=a+b", expected: Sca, a: 6, b: 4, c: 5},
+	{description: "all sides zero", expected: NaT, a: 0, b: 0, c: 0},
+	{description: "negative length", expected: NaT, a: 3, b: 4, c: -5},
+	{description: "not a triangle", expected: NaT, a: 1, b: 1, c: 3},
+	{description: "not a triangle", expected: NaT, a: 2, b: 5, c: 2},
+	{description: "not a triangle", expected: NaT, a: 7, b: 3, c: 2},
```

Comment:
Could you add the values to the name here as well like in line 20 to make the test names distinct?